src/components/team/TeamCard.tsx

```diff
@@ -0,0 +1,70 @@
+/* eslint-disable @next/next/no-img-element */
+import Link from "next/link";
+import { Router, useRouter } from "next/router";
+import React from "react";
+import { TeamCardI } from "./team-interfaces";
+
+const TeamCard: React.FC<TeamCardI> = ({
+	image,
+	name,
+	position,
+	description,
+	socials,
+	email,
+	phone,
+	address,
+}) => {
+	const router = useRouter();
+	const { facebook_url, twitter_url } = socials;
+	// function sends targeted data to targetpath using nextRouter
+	function sendSingleTeamProp() {
```

Comment:
whatever you are doing here is wrong

Reply:
I am not clear on which is wrong sir.

Parsing 'socials' into the use-router push query was giving errors so I destructured them individually before parsing. 

The function was used to push out the unique properties held by a particular team card into the single team page. Function gets triggered when user clicks on the team member name.